02_calculator/calculator.rb

```diff
@@ -0,0 +1,31 @@
+def add(a,b)
+  a+b
+end
+
+def subtract(a,b)
+  return (a-b)
```

Comment:
You don't need `( ...  )`
And you don't need `return` -- ruby returns by default the result of the last line of code in the function. Should be similar to what you did for the `add` function